### PR TITLE
fix:app bar title

### LINKED ISSUE #1620: Using string resource for app title
**Actual Behaviour**

Some of the layout files do not use string.xml for app title

**Expected Behaviour**

The layout files should follow the naming conventions of using string.xml

**Steps to reproduce it**

Using string.xml

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by riyasingh1004**
Fixes #1620 

**Changes**: 

Using string resources for multimeter and Logic Analyzer appbar title

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

### CODE REVIEW

**Comment by cweitat** [comment]
@riyasingh1004 pls ensure travis is passed

**Comment by riyasingh1004** [comment]
@cweitat I don't understand why it's failing. Can you please help .

**Comment by cweitat** [comment]
@riyasingh1004 read the log. the error message is there

File: app/src/main/res/layout/activity_logic_analyzer.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
         android:layout_height="wrap_content"
         android:background="@color/colorPrimary"
         android:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar"
-        app:title="Logical Analyzer" />
+        app:title="@string/logical_analyser" />
```
Comment (CloudyPadmal) [inline app/src/main/res/layout/activity_logic_analyzer.xml]: Such a string resource is not found according to Travis build log

File: app/src/main/res/layout/activity_multimeter.xml [hunk 2]
```diff
@@ -18,7 +18,7 @@
             android:layout_height="?attr/actionBarSize"
             android:background="?attr/colorPrimary"
             app:popupTheme="@style/AppTheme.PopupOverlay"
-            app:title="Multimeter" />
+            app:title="@string/multimeter" />
```
Comment (CloudyPadmal) [inline app/src/main/res/layout/activity_multimeter.xml]: Similarly here. Please double check if the resource is there and try running the updated code in your PC to see if it's working before making a PR.

**Comment by ho-dor** [comment]
Can I take this up . There has not been any update in this PR for some time
.

**Comment by CloudyPadmal** [comment]
Hello @ho-dor , we'll follow up on this in a day or two

**Comment by ho-dor** [comment]
@CloudyPadmal this PR has also been idle . Can I take it up . Thanks .

**Comment by cweitat** [comment]
@ho-dor  go ahead

**Comment by Smit2010** [comment]
Is anyone has taken up this? If not I would like to take this up.
@CloudyPadmal

**Comment by ho-dor** [comment]
I am working on it. Sorry for the delay . Been working on high priority
tagged issues . Thanks.

Regards,
Kunal Rai(BT16CSE058),
NATIONAL INSTITUTE OF TECHNOLOGY , UTTARAKHAND,
+91-7895780897

**Pull request closed without merging**